src/EssentialsPE/BaseFiles/BaseTask.php

```diff
@@ -5,7 +5,7 @@
 namespace EssentialsPE\BaseFiles;
 
 use EssentialsPE\Loader;
-use pocketmine\scheduler\Plugin;
```

Comment:
lol

Reply:
A typo I introduced in previous commits. Realized it when removing the updater